src/PagesBody/LeaderboardPage/LeaderboardPage.js

```diff
@@ -1,9 +1,21 @@
 import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import LeaderboardList from './../../Molecules/LeaderboardList/LeaderboardList';
+import MainButtonStyled from './../../Atoms/MainButton/MainButton';
+import { useParams } from 'react-router-dom';
 
 const LeaderboardPage = () => {
+  const { contestId, classId } = useParams();
   return (
     <ColumnWrapper paddingLeftRight={1}>
-      <h1>I am leaderboard page</h1>
+      <h1>
+        <br></br> <br></br>I am Past Contest leaderboard (summary of class
+        {classId}) page
+      </h1>
+      <LeaderboardList contestId={contestId} classId={classId} />
+      <div>
+        <br></br>
+      </div>
+      <MainButtonStyled primary text="drukuj protokół" />
     </ColumnWrapper>
   );
 };
```

Comment:
you can put navigation to that page at "Lista klas" Page, at the button / fake button at the bottom of that page.
and "Drukuj protokół" should be just when it is a flow for staff, not for particiapnts (when they have just possibilit to see results)